docs/guides/tutorials/nextjs_app_router.rst

```diff
@@ -0,0 +1,627 @@
+.. _ref_guide_nextjs_app_router:
+
+====================
+Next.js (App Router)
+====================
+
+:edb-alt-title: Building a simple blog application with
+   EdgeDB and Next.js (App Router)
+
+We're going to build a simple blog application with
+`Next.js <https://nextjs.org/>`_ and EdgeDB. Let's start by scaffolding our
+app with Next.js's ``create-next-app`` tool.
+
+.. code-block:: bash
+
+  $ npx create-next-app@latest
+
+You'll be prompted to provide a name (we'll use ``nextjs-blog``) for your 
+app and choose project options. For this tutorial, we'll go with the 
+recommended settings including TypeScript, App Router, and 
+**opt-ing out** of the ``src/`` directory.
```

Comment:
> …we'll go with the recommended settings…

Since we don't know for certain all these options will continue to be the recommended settings in perpetuity, I think we should delineate each of the settings we want the user to select. We've picked out a few of them here and called them out, but I think we're leaving the rest for the user to choose the "recommended setting" which could change in the future.

Here are the prompts I answered with the ones we haven't called out in bold:

> ✔ Would you like to use TypeScript? … No / Yes
> ✔ **Would you like to use ESLint? … No / Yes**
> ✔ **Would you like to use Tailwind CSS? … No / Yes**
> ✔ Would you like to use `src/` directory? … No / Yes
> ✔ Would you like to use App Router? (recommended) … No / Yes
> ✔ **Would you like to customize the default import alias (@/*)? … No / Yes**

Reply:
I'll add a code snippet, highlighting the options we're picking for this particular tutorial.